compiler/rustc_target/src/abi/call/bpf.rs

```diff
@@ -0,0 +1,31 @@
+// see BPFCallingConv.td
```

Comment:
This is not all that useful a reference to a typical Rust developer who doesn't dabble in LLVM. Could this perhaps be replaced with a link to this file?

Reply:
✅